examples/notebooks/dev_ray_submission.ipynb

```diff
@@ -110,11 +109,28 @@
     "# Submit the job to the Ray cluster\n",
     "# Note: LM Buddy is invoked by 'python -m lm_buddy run ...' since the CLI is not installed in the environment\n",
     "client.submit_job(\n",
-    "    entrypoint=f\"python -m lm_buddy run simple --config simple_config.yaml\",\n",
+    "    entrypoint=f\"python -m lm_buddy run simple --config simple/simple_config.yaml\",\n",
     "    runtime_env=runtime_env,\n",
     ")"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "# Alternate submission where you're running a quantized fine-tune job\n",
+    "client.submit_job(\n",
+    "    entrypoint=f\"python -m lm_buddy run simple --config evaluation/lm_harness_quantized_config.yaml\",\n",
+    "    runtime_env=runtime_env,\n",
+    "    entrypoint_num_gpus=1\n",
+    ")"
+   ],
+   "metadata": {
+    "collapsed": false
+   },
+   "id": "b2ab4f17defeb574"
+  },
```

Comment:
I don't think this is helpful for the `dev_ray_submission` context.

if anything, we should add explicit example notebooks for each job type.

Reply:
Looks like [Mistral just released](https://github.com/mistralai/cookbook) a cookbook, too

Reply:
For instance, an example "evaluating Mistral 7b across multiple axis", where we go through finetuning Mistral 7B and evaluating it on lm-harness, ragas, and prometheus, and compare and contrast results.

Reply:
Yeah, that would be great - let's add it to the agenda.